### PR TITLE
Feature/paypal incontext

### PR DESCRIPTION
**Pull request by slogsdon**

### CODE REVIEW

File: classes/class-wc-gateway-securesubmit-paypal.php [hunk 1]
```diff
@@ -88,204 +137,129 @@ public function is_valid_for_use() {
...
-            if (function_exists('wc_add_notice')) {
-                wc_add_notice($error, 'error');
-            } else {
-                $woocommerce->add_error($error);
-            }
-            return false;
-        }
+    public function process_refund($orderId, $amount = null, $reason = '')
+    {
+        return $this->refund->call($orderId, $amount, $reson);
```
Comment (SenthilKumar-86) [inline classes/class-wc-gateway-securesubmit-paypal.php]: update $reson as $reason

Comment (slogsdon) [inline classes/class-wc-gateway-securesubmit-paypal.php]: nice catch. will update

File: classes/wc-gateway-securesubmit-paypal/class-refund.php [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        if ($response->responseCode == "00") {
+            $reason = $reason == '' ? '' : '. Reason for refund: '.$reason;
+            $order->add_order_note(
+                __('Heartland PayPal refund completed. Transaction id: '
+                    . $response->transactionId . $reason, 'wc_securesubmit')
+            );
+            return true;
+        }
+
+        call_user_func(arra($this->parent, 'debugLog'), 'Refund Failed in Portico call with responseCode ' . $response->responseCode);
```
Comment (SenthilKumar-86) [inline classes/wc-gateway-securesubmit-paypal/class-refund.php]: update "arra(" as "array("

**Comment by slogsdon** [comment]
@SenthilKumar-86 corrected. These were remnants of an older logging setup.

**Pull request merged**